python/llm/src/ipex_llm/transformers/utils.py

```diff
@@ -269,7 +269,8 @@ def module_name_process(full_module_name):
     return new_module_name, layer, cur_module
 
 
-def get_cur_qtype_and_imatrix(qtype, full_module_name, imatrix_data, model_config=None):
+def get_cur_qtype_and_imatrix(qtype, full_module_name, imatrix_data, model_config=None,
+                              mixed_precision=False):
```

Comment:
is it used?

Reply:
Not used now, but I guess maybe will be used in the future : )

Shall I remove it now ?

Reply:
yes, remove it for now

Reply:
Done.